CHANGELOG.md

```diff
@@ -9,6 +9,11 @@ The format is based on [Keep a Changelog][kc], and this project adheres to
 
 ## [Unreleased]
 
```

Comment:
I don't think this trailing newline should be here, strictly speaking, according to KeepAChangelog.